apps/etterlatte-saksbehandling-ui/client/src/components/behandling/beregningsgrunnlag/Beregningsgrunnlag.tsx

```diff
@@ -76,6 +77,32 @@ const Beregningsgrunnlag = () => {
               {behandling.virkningstidspunkt ? formaterStringDato(behandling.virkningstidspunkt.dato) : 'ukjent dato'}
             </strong>
           </BodyShort>
+        </HeadingWrapper>
+      </ContentHeader>
+      <TrygdetidWrapper>
+        <Soeknadsvurdering
+          tittel={'Trygdetid'}
+          hjemler={[
+            { tittel: '§ 3-5 Trygdetid ved beregning av ytelser', lenke: 'https://lovdata.no/lov/1997-02-28-19/§3-5' },
+          ]}
+          vurderingsResultat={null}
+          status={null}
+        >
+          <TrygdetidInfo>
+            <p>
+              Trygdetiden er minst 40 år som følge av faktisk trygdetid og fremtidig trygdetid. Faktisk trygdetid er den
+              tiden fra avdøde fylte 16 til personen døde. Fremtidig trygdetid er tiden fra dødsfallet til året avdøde
+              hadde blitt 66 år. Saksbehandler bekrefter at følgende stemmer for denne behandlingen ved å gå videre til
+              beregning:
+            </p>
+            <p>
+              Trygdetid: <strong>40 år</strong>
```

Comment:
Skal ikke koble den til den hardkodede beregningen fra backend så det oppdateres automatisk her hvis backend endrer på det?

Reply:
Enig, men vi får ikke gjort det (nå) ettersom det ikke er data noe sted. Det er hardkodet inn i beregningsperioder, men på dette steget er ikke beregningsperioder generert enda 🤷 
Men det finnes to jiraer som sørger for at hardkodingen ikke forblir. En på å faktisk håndtere trygdetid som ikke er 40 år (https://jira.adeo.no/browse/EY-981) og en for å flytte søskenjustering-data til ny tabell i beregningsdatabasen (https://jira.adeo.no/browse/EY-1618). Da vil man også lagre trygdetid der